src/utils/validators.js

```diff
@@ -150,69 +150,81 @@ export class UserValidator extends BaseValidator {
 }
 
 export class ServerValidator extends BaseValidator {
-  static validateServerName = (_name) => {
+  static validateCreationInfo = (_name, _description) => {
     const name = this.validateString(_name, "name");
-    const regex = /^[a-z0-9]{3,20}$/i;
-    if (!regex.test(name)) {
+    if (!name.match(/^[a-z0-9]{3,20}$/i)) {
       throw new BadRequestError(
-        400,
-        this.validateServerName.name,
-        "Server name must be between 3 and 20 alphanumeric characters."
-      );
+      400,
+      this.validateCreationInfo.name,
+      'Name must be between 3 and 20 alphanumeric characters.');
     }
-    return name;
-  };
 
-  static validateDescription = (_description) => {
-    if (typeof _description !== "string") {
+    const description = this.validateString(_description, "description");
```

Comment:
Doing this would make the server throw a 400 error since descriptions would now be required. Please just run the program yourself and test everything before committing so we don't have go back later and try to find where stuff broke.